music21/sorting.py

```diff
@@ -129,10 +129,13 @@ def __gt__(self, other):
             elif self.atEnd == 1:
                 return False
             else:
-                return self.offset > other
+                return (self.offset > other)
         except ValueError:
             return NotImplemented
 
+    def __ne__(self, other):
```

Comment:
thanks!

Reply:
I didn't add this method, or make some of the changes that produced the linting errors. I'm not sure where some of these diff lines came from. I do understand all your comments though, so I'm making the necessary changes now.